modin/engines/base/io/file_dispatcher.py

```diff
@@ -117,12 +139,31 @@ def file_size(cls, f):
         return size
 
     @classmethod
-    def file_exists(cls, file_path):
+    def file_exists(cls, file_path: str, glob_path: bool = False) -> bool:
```

Comment:
I think I would prefer a different utility specific to globs here instead of a flag.

Reply:
fixed